oak_functions_app/README.md

```diff
@@ -0,0 +1,4 @@
+# Oak Functions Restricted Application
```

Comment:
I have called them "Trusted Application". 

I am happy to call them "Restricted Applications" too, with a slight preference to "Trusted" as in: "Trusted Applications running on the Restricted Kernel".

What do you and others think?

Reply:
According to my suggested terminology this would be "Oak Functions Enclave Application"